docs/ABOUT.md

```diff
@@ -1 +1,12 @@
-[TypeScript](http://www.typescriptlang.org/) is a language for application-scale JavaScript. TypeScript adds optional types, classes, and modules to JavaScript. TypeScript supports tools for large-scale JavaScript applications for any browser, for any host, on any OS. TypeScript compiles to readable, standards-based JavaScript. Try it out at the [playground](http://www.typescriptlang.org/Playground), and stay up to date via [our blog](https://blogs.msdn.microsoft.com/typescript) and [Twitter account](https://twitter.com/typescriptlang).
\ No newline at end of file
+[TypeScript](http://www.typescriptlang.org/) (TS) is a superset of JavaScript (JS), created at Microsoft
+in response to frustration developing large-scale applictions in JS.
+It is currently an open-source project hosted on Github.
+It supports tools for any browser as well as Node, for any host, on any OS. TTS compiles to readable, standards-based JavaScript.
+
+TS adds a flexible type system to JS, in addition to interfaces (custom types) and modifying the syntax of some ECMAScript features such as classes. It supports many upcoming
+ECMASCript features (such as async/await). Types allow tooling available in most code editors
+that improve the development experience such as code completion and method detection, both in
+your own code and in packages you use. It is compatible with all existing JS packages.
+TS transpiles to clean, readable JS.
+
+Try it out at the [playground](http://www.typescriptlang.org/Playground), and stay up to date via [our blog](https://blogs.msdn.microsoft.com/typescript) and [Twitter account](https://twitter.com/typescriptlang).
```

Comment:
This is not Exercism's blog and Twitter account. Maybe _the Typescript blog and Twitter account_?